src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -1045,10 +1048,24 @@ def export_group_as_template(
         export_options.append('IncludeComments')
     if include_parameter_default_value:
         export_options.append('IncludeParameterDefaultValue')
+    if skip_resource_name_params:
+        export_options.append('SkipResourceNameParameterization')
+    if skip_all_params:
+        export_options.append('SkipAllParameterization')
+
+    resources = []
+    if resource_ids is None or resource_ids == "*":
```

Comment:
>resource_ids [](start = 31, length = 12)

just to confirm here if it should be resource_ids[0] == "*"?

Reply:
updated the code.